### ISSUE TITLE
Refresh the order of content on the main page

### ISSUE THREAD

**Issue by sharadagarwal** [sharadagarwal/ACM-SIGMOBILE-Craft-Website#2]
Suman has suggested the following ordering:

1.	Awards - Test of Time winners (big box)
2.	Events - HotMobile 2017
3.	Publications — MobiCom 2016 best paper award winner
4.	Publications — MobiSys 2016 best paper award winner
5.	Events - Call for participation of SenSys 2016?
6.	Awards — Rockstar — Karthik S. (big box)
7.	Publications — a paper from GetMobile October 2015 issue
8.	Awards — SIGMOBILE research highlights 2016 (two MobiSys papers)
9.	Events — MobiSys 2017
10.	Events — Fashionabil.IT 2016 (pointer to this event)
11.	Events — MobiHoc 2017

**Comment by kjayarajah** [comment]
The ordering on the landing page depends on the date the post was made and the explicitly set priority (1- for the big box, 2-for the mid-size box). Figured that the "post date" of each entry, separate from the "created/modified date", is actually editable -- so I was able to back-date some of the older posts (e.g., SenSys 2013 which was recently added) and keep the current ones to appear in the landing page.
-----
Unfortunately, I'm not seeing this consistently on the other pages; recent posts are at the end of the list -- Events -> MobiHoc, Events -> MobiSys, Events -> SenSys
-----
This entry is still missing -- might you have any pointers to this?
10. Events — Fashionabil.IT 2016
